README.md

```diff
@@ -70,7 +70,7 @@ services:
       PLUGINS: >-
         academic-bloggers-toolkit,
         co-authors-plus,
-        [WP-API-master]https://github.com/WP-API/WP-API/archive/master.zip,
+        [WP-API]https://github.com/WP-API/WP-API/archive/master.zip,
```

Comment:
The need for `-master` was revoked a few versions ago with WP-CLI. They now correct for this internally.